### PR TITLE
Drawer - block frame update for missing items

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Drawer - block frame update for missing items.

## Changelog
Drawer - block frame update for missing items.

### CODE REVIEW

**Comment by Inbal-Tish** [comment]
@lidord-wix Please take another look

File: src/components/drawer/Swipeable.tsx [hunk 1]
```diff
@@ -243,6 +243,14 @@ export default class Swipeable extends Component<Props, StateType> {
...
   };
 
+  _hasLeftActions() {
+    return this.props.renderLeftActions !== undefined;
+  }
+
+  _hasRightActions() {
+    return this.props.renderRightActions !== undefined;
+  }
+
```
Comment (lidord-wix) [inline src/components/drawer/Swipeable.tsx:253]: Why as functions and not as consts?

**Review by lidord-wix — COMMENTED** [review]
Works now!
consider changing the functions to regular consts

File: src/components/drawer/Swipeable.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/drawer/Swipeable.tsx]: habit... Done

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**